tests/e2e/mapper/mapper_suite_test.go

```diff
@@ -0,0 +1,246 @@
+package mapper
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
+	"os/exec"
+	"path"
+	"testing"
+	"time"
+
+	v1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/api/core/v1"
+
+	"github.com/kubeedge/kubeedge/tests/e2e/utils"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+//context to load config and access across the package
+var (
+	ctx            *utils.TestContext
+	cfg            utils.Config
+	nodeSelector   string
+	nodeName       string
+	chconfigmapRet = make(chan error)
+)
+
+var (
+	runController       = "cd ${GOPATH}/src/github.com/kubeedge/kubeedge/cloud; sudo nohup ./edgecontroller > edgecontroller.log 2>&1 &"
```

Comment:
Please check with @pavan187 , we need to get rid of these commands and put them in scripts and then call the scripts.

Reply:
Updated

Reply:
Updated